add_mozilla_tag.py

```diff
@@ -1,11 +1,11 @@
 from pelican import signals
+from pelican.contents import Tag
 
 def add_category(generator):
+    tag = Tag('mozilla', generator.settings)
     for article in generator.articles:
-        if hasattr(article, 'tags'):
-            article.tags.append('mozilla')
-        else:
-            setattr(article, 'tags', ['mozilla'])
+        if not hasattr(article, 'tags'):
+            setattr(article, 'tags', [tag,])
```

Comment:
I've also changed the behaviour to only add the "mozilla" tag if nothing else is specified.